extensions/2.0/Khronos/KHR_animation_pointer/README.md

```diff
@@ -0,0 +1,318 @@
+| Property                    | Animatable |
+|:----------------------------|:-----------|
+| `offset`                    | ✅        |
+| `rotation`                  | ✅        |
+| `scale`                     | ✅        |
+| `texCoord`                  | ❌        |
+
+#### [KHR_xmp_json_ld](https://github.com/KhronosGroup/glTF/tree/main/extensions/2.0/Khronos/KHR_xmp_json_ld)
+❌ No animatable properties.
+
+### Example target templates for extensions
+
+| `pointer`                                                                                       | Accessor Type | Component Type(s)   | Description              |
+|:------------------------------------------------------------------------------------------------|:--------------|:--------------------|:-------------------------|
+| `"/materials/{}/pbrMetallicRoughness/baseColorTexture/extensions/KHR_texture_transform/offset"` | `"VEC2"`      | `5126`&nbsp;(FLOAT) | XY offset vector         |
+| `"/extensions/KHR_lights_punctual/lights/{}/intensity"`                                         | `"SCALAR"`    | `5126`&nbsp;(FLOAT) | Light intensity          |
+| `"/materials/{}/extensions/KHR_materials_emissive_strength/emissiveStrength"`                   | `"SCALAR"`    | `5126`&nbsp;(FLOAT) | Emissive strength factor |
+
+## glTF Schema Updates
+
+* **JSON schema**: [glTF.KHR_animation_pointer.schema.json](schema/glTF.KHR_animation_pointer.schema.json)
+
+## Notes and clarifications
+
+_This section is non-normative._
+
+### JSON Pointer
+
+For a `SCALAR` value, the JSON Pointer targets the glTF property.  
+This means, the property is replaced by the interpolated value.
+  
+For any other case, the JSON Pointer targets the glTF property as well but **must** be an `array` property. For clarification, it is not allowed to animate a single element in an `array`.  
+This means, that the two or more elements do replace the values in the array.  
+
+### Material Animation
+
+Material animation targets materials directly. This means that when animating the material, all nodes using that material are updated together. If individual material animation per node is desired, each of them needs to have its own material to animate.
+
+### Mesh Animation
+
+Mesh animation targets meshes directly. This means that when animating the mesh, all nodes using that mesh are updated together. This is the case, when the `weights` are animated.
+
+## Examples
+
+The following snippet shows the changes for [`animations`](https://github.com/KhronosGroup/glTF-Sample-Models/blob/master/2.0/AnimatedCube/glTF/AnimatedCube.gltf#L117) from the [AnimatedCube](https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/AnimatedCube) asset.
+
+```javascript
+"animations" : [
+    {
+        "channels" : [
+            {
+                "name" : "Targeting x, y, z, w for a rotation of node at index 0."
+                "sampler" : 0,
+                "target" : {
+                    "path" : "pointer"
```

Comment:
This is not valid glTF. You can't use `"pointer"` here inside of `"path"`.

The [Animation Channel Target schema](https://github.com/KhronosGroup/glTF/blob/main/specification/2.0/schema/animation.channel.target.schema.json) explicitly only allows `"translation"`, `"rotation"`, `"scale"`, and `"weights"` as values for `"path"`.

Any new values need to exclusively be present inside of the extension. Take a look at how [`EXT_texture_webp`](https://github.com/KhronosGroup/glTF/tree/main/extensions/2.0/Vendor/EXT_texture_webp#using-without-a-fallback) works. For example, this could work:

```json
"target": {
    "extensions": {
        "KHR_animation_pointer": {
            "pointer": "/nodes/0/rotation"
        }
    }
}
```

Reply:
As far as I'm aware "anyOf" allows extensions to add future string values, and implementations are supposed to ignore ones they don't understand. New fields need to go into an extension.

See https://github.com/KhronosGroup/glTF/pull/899#discussion_r110480173

cc @lexaknyazev, would appreciate your input.

Reply:
I already brought that up in https://github.com/KhronosGroup/glTF/pull/2033#issuecomment-1104262883 (see bottom of that comment), with some responses by Ed and Don, but maybe Alexey still wants to chime in here.

Reply:
I apologize, I was mistaken. Ed is correct, one of the values is `{ "type": "string" }` so this is fine. I would click the resolve conversation button but it seems I don't have that button.